_data/education2.yml

```diff
@@ -0,0 +1,9 @@
+- course: Software Developer 
+  institute: Microverse
+  dates: 2020 - 2021
+  logo: uob.png
```

Comment:
You could use this image for the school logo
![microverse](https://media-exp1.licdn.com/dms/image/C560BAQHr8P7gQ95yCQ/company-logo_200_200/0/1578673850004?e=1635379200&v=beta&t=lcOlWKQuwFd5bpmDh3NJkZmNZXD7QC6KE8SKuVZJ0ok)